slow_odgi/mygfa.py

```diff
@@ -122,7 +122,8 @@ def __str__(self):
             "P",
             self.name,
             ",".join(f"{n}{'+' if o else '-'}" for (n, o) in self.segments),
-            ",".join(str(a) for a in self.overlaps) if self.overlaps else "*",
```

Comment:
ODGI discards overlap info somewhere along the way, so I needed to throw out this information in order to match with ODGI's outputs.